ynput/ayon-core
# Maya: Improve validate current renderlayer renderable artist report

## Changelog Description

Improve validate current renderlayer renderable artist report

## Additional info

n/a

## Testing notes:

1. Set up a render scene
2. Set up no renderable camera in the current renderlayer
3. Set up a renderable camrea in another renderlayer (the one you are submitting)
4. Publish (submit scene)

Validation should be clear to the artist.
